examples/cxxopts/CMakeLists.txt

```diff
@@ -9,8 +9,11 @@ include("../common.cmake")
 
 project(download-cxxopts)
 
+
 hunter_add_package(cxxopts)
 find_package(cxxopts CONFIG REQUIRED)
+string(REPLACE "." ";" VERSION_LIST ${HUNTER_cxxopts_VERSION})
+list(GET VERSION_LIST 0 CXXOPTS_MAJOR)
```

Comment:
I don't see the usage of `CXXOPTS_MAJOR` variable, can it be removed?

Reply:
Oh, I forgot to remove that variable. 
I will remove it.